cli/internal/daemonclient/daemonclient.go

```diff
@@ -32,12 +34,32 @@ func New(client *connector.Client) *DaemonClient {
 	}
 }
 
+// formats a repo-relative glob to unix format with ':' characters handled.
+// On windows, ':' is an invalid path character, but you can, and Turborepo does,
+// read to and write from files that contain alternate data streams denoted by ':'.
+// In the case of windows and an alternate data stream, we want change notifications just
+// for the root file. Note that since ':' denotes a data stream for a _file_, it cannot
+// appear in a directory name. Thus, if we find one, we know it's in the filename.
+// See https://learn.microsoft.com/en-us/sysinternals/downloads/streams
```

Comment:
I'm hoping you can't delete data streams from a file individually. i.e. Do we need to worry about the situation where `build:schema` and `build:types` point to the same log file, but only the `types` stream gets deleted and we miss it's removal since the file is still there?

Reply:
Based on the `-d` flag of [the CLI tool](https://learn.microsoft.com/en-us/sysinternals/downloads/streams#using-streams) I'm guessing the answer is yes. Obviously non-blocking, but fun.

Reply:
You can delete individual streams, we get a modification event for the file, which is sufficient for our purposes.